docs/metrics/prometheus/README.md

```diff
@@ -152,7 +152,7 @@ MinIO Gateway instances enabled with Disk-Caching expose caching related metrics
 MinIO Gateway instance exposes metrics related to Gateway communication with the cloud backend (S3, Azure & GCS Gateway).
 
 - `gateway_<gateway_type>_requests`: Total number of GET & HEAD requests made to cloud backend. This metrics has a label `method` that identifies GET & HEAD Requests.
-- `gateway_<gateway_type>_bytes_sent`: Total number of bytes sent to cloud backend (in GET & HEAD Requests).
+- `gateway_<gateway_type>_bytes_sent`: Total number of bytes sent to cloud backend (in PUT & POST Requests).
```

Comment:
can we document the new values?

Reply:
done - the additional counts will be distinguished by `method` for `gateway_<gateway_type>_requests`